core/src/i18n/en.yaml

```
@@ -106,48 +106,52 @@ brokers:
     url: URL
   title: Brokers
 btp-instances:
-  create:
-    external-name-placeholder: Defaults to the instance name if empty.
-    offering-name-description: SAP BTP service offering name.
-    params-invalid: Parameters must be an object.
-    plan-name-description: The plan name to use for the ServiceInstance.
-    title: Create $t(btp-instances.resource-type).
   external-name: External Name
   instance-data: ServiceInstance Data
   instance-id: Instance ID
+  messages:
+    params-invalid: Parameters must be an object.
```

```suggestion
    params-invalid: Parameter invalid; must be an object.
```
I was struggling with the plural `Parameters` vs singular `object`.  Grzegorz explained it makes sense but maybe we can find another way to phrase it?